src/main/java/com/codeborne/selenide/commands/Click.java

```diff
@@ -15,76 +14,27 @@
 
 @ParametersAreNonnullByDefault
 public class Click implements Command<Void> {
-  private final FileContent jsSource = new FileContent("click.js");
 
   @Override
   @Nullable
   public Void execute(SelenideElement proxy, WebElementSource locator, @Nullable Object[] args) {
     Driver driver = locator.driver();
     WebElement webElement = locator.findAndAssertElementIsInteractable();
-
+    ClickOptions clickOptions;
     if (args == null || args.length == 0) {
-      click(driver, webElement);
-    }
-    else if (args.length == 1) {
-      ClickOptions clickOptions = firstOf(args);
-      click(driver, webElement, clickOptions);
-    }
-    else if (args.length == 2) {
-      Integer offsetX = firstOf(args);
-      Integer offsetY = (Integer) args[1];
-      click(driver, webElement, offsetX, offsetY);
-    }
+      clickOptions = driver.config().clickViaJs()
+        ? ClickOptions.usingJavaScript()
+        : ClickOptions.usingDefaultMethod();
+    } else if (args[0] instanceof Integer) {
+      int offsetX = (Integer) args[0];
+      int offsetY = (Integer) args[1];
+      clickOptions = driver.config().clickViaJs()
+        ? ClickOptions.usingJavaScript(offsetX, offsetY)
+        : ClickOptions.usingDefaultMethod(offsetX, offsetY);
+    } else {
+      clickOptions = firstOf(args);
+    }
+    clickOptions.click(driver, webElement);
     return null;
   }
-
-  protected void click(Driver driver, WebElement element) {
-    if (driver.config().clickViaJs()) {
-      click(driver, element, 0, 0);
-    }
-    else {
-      element.click();
```

Comment:
Do we want to remove WebElement click()? Is it the same as Actions click()?

Reply:
Fixed a bug by creating `ClickOptions.ZeroOffsetDefaultMethod` class